languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala

```diff
@@ -68,16 +68,7 @@ class WdlDraft2LanguageFactory(override val config: Config) extends LanguageFact
     }
 
     def validationCallable = new Callable[ErrorOr[WdlNamespaceWithWorkflow]] {
-      def call: ErrorOr[WdlNamespaceWithWorkflow] = source match {
-        case w: WorkflowSourceFilesWithDependenciesZip =>
-          for {
-            importsDir <- LanguageFactoryUtil.validateImportsDirectory(w.importsZip)
-            wf <- WdlNamespaceWithWorkflow.load(workflowSource, importResolvers map resolverConverter).toErrorOr
-            _ = importsDir.delete(swallowIOExceptions = true)
-          } yield wf
-        case _: WorkflowSourceFilesWithoutImports =>
-          WdlNamespaceWithWorkflow.load(workflowSource, importResolvers map resolverConverter).toErrorOr
-      }
+      def call: ErrorOr[WdlNamespaceWithWorkflow] = WdlNamespaceWithWorkflow.load(workflowSource, importResolvers map resolverConverter).toErrorOr
```

Comment:
This change is from a different PR that is also merged, right?

It's great to have, thanks for thinking of it.

Reply:
Update:

There have been a number of changesets in this area, I got confused. This PR eliminates code that unzips the imports, does nothing with them, and then cleans them up.

The PR that really would help Green Team is this one, which cleans up the unzip directories we actually use, and currently fail to clean up: https://github.com/broadinstitute/cromwell/pull/4581

Reply:
I'll close this and re-open with the correct changes picked from #4581